src/routes/add-place.ts

```diff
@@ -34,6 +34,7 @@ export default async function addPlace(fastify: FastifyInstance) {
   fastify.post('/place/dob', async (req, resp) => {
     const { place_type, prefix, prop_type } = req.query as any;
     const contactType = Config.getContactType(place_type).contact_properties.find(prop => prop.type === prop_type);
+
```

Comment:
can you revert these white space changes?  They just add clutter to the PR

Reply:
this file `add-place.ts` should not be in this PR because you just made white space changes - please revert.

Reply:
can you revert these white space changes?